agent/dockerclient/dockerapi/errors.go

```diff
@@ -30,6 +30,8 @@ const (
 	CannotDescribeContainerErrorName = "CannotDescribeContainerError"
 	// CannotGetContainerTopErrorName is the name of the top container error.
 	CannotGetContainerTopErrorName = "CannotGetContainerTopError"
+	// DockerExecTimeoutErrorName is the name of docker timeout error.
+	DockerExecTimeoutErrorName = "DockerExecTimeoutError"
```

Comment:
is this specific to exec create API or other exec ones too?

Reply:
Removed this. I think we can reuse DockerTimeoutError